src/main/java/com/cloudant/client/api/CloudantClient.java

```diff
@@ -506,6 +525,10 @@ Response put(URI uri, Object object, boolean newEntity, int writeQuorum, Gson gs
                 h.put("scheme", uri.getScheme());
                 h.put("hostname", uri.getHost());
                 h.put("port", new Integer(uri.getPort()).toString());
+                if(uri.getUserInfo() != null && !uri.getUserInfo().isEmpty()) {

```

Comment:
Why is this added here? I thought this already happened in `HttpConnection`'s execute method.